src/game.py

```diff
@@ -5,7 +5,8 @@ def hello_world(self):
     def display_welcome_message(self):
         print("Welcome to Tic Tac Toe")
 
-
-if __name__ == "__main__":
-    game = Game()
-    game.display_welcome_message()
+    def display_board(self):
+        board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
```

Comment:
`display_board` appears to both initiate a board and print it.  This is doing two things.  We want our functions to do one thing and do it well.  What do you think about initializing the `board` somewhere else, and then passing it in as a parameter?

Reply:
Yes that makes sense.  I will implement that!